Forms/index.html

```diff
@@ -9,82 +9,96 @@
 </head>
 
 <body>
-    <img src="res/left-img.png" class="img1">
-    <h1>
-        <font size="15" face="segoe print">Алко.net</font>
-    </h1>
+    <div class="head-container">
+        <h1>
+            <font size="15" face="segoe print">Алко.net</font>
+        </h1>
+        <img src="res/left-img.png" class="img1">
+    </div>
 
     <div class="container">
-        <form action="#" method="get" enctype="multipart/form-data" id="form">
-            <div class="first_collum">
+        <div class="first_column">
+            <div>Текст </div>
+        </div>
+        <form id="form">
+            <div class="second_column">
                 <p>
                 <h2>Мы проводим анкетирование, пожалуйста пройдите опрос</h2>
                 </p>
                 <div class="text_form">
                     <p>Введите имя</p>
-                    <input placeholder="Иванов Иван" type="text" name="username" value=""><!--тег disabled блокирует поле-->
+                    <input placeholder="Иванов Иван" type="text" name="username"><!--тег disabled блокирует поле-->
                 </div>
                 <div class="text_form">
                     <p>Введите email</p>
-                    <input tabindex="1" placeholder="simple_email@mail.com" type="text" name="userpass" value=""><!-- tabindex на скоьлко пунктов прескочит формы нажав tab-->
+                    <input tabindex="1" placeholder="simple_email@mail.com" type="text" name="userpass"><!-- tabindex на скоьлко пунктов прескочит формы нажав tab-->
```

Comment:
Yup, that's how tabindex works, and it's great that you know that it exists. But what is it here for?

Reply:
It's from guide from YT after i del this